velocity/tool/src/java/org/sakaiproject/cheftool/menu/MenuDivider.java

```diff
@@ -133,9 +133,9 @@ public String getForm()
 	 * 
 	 * @return The sub-items of the item.
 	 */
-	public List getItems()
+	public List<MenuItem> getItems()
 	{
-		return new Vector();
+		return new Vector<>();
```

Comment:
Can we just get rid of the deprecated `Vector` all together and return a `new ArrayList<>()`?. Same goes for all the other instances of `Vector` touched in the PR